model/client.go

```diff
@@ -24,4 +26,9 @@ type Client struct {
 	// required: true
 	// example: Android Phone
 	Name string `gorm:"type:text" form:"name" query:"name" json:"name" binding:"required"`
+	// The last time the client was seen.
```

Comment:
Can we also update last seen when the token was used for api calls. I think the update could be done in auth/authentication.go in the RequireClient call. Similar to what I described in this comment: https://github.com/gotify/server/issues/400#issuecomment-1247082718